website/source/docs/builders/vmware-iso.html.md.erb

```diff
@@ -98,6 +98,9 @@ builder.
     five seconds and one minute 30 seconds, respectively. If this isn't
     specified, the default is `10s` or 10 seconds.
 
+-   `cpus` (number) - The number of cpus to use when building the VM.
+    Defaults to building with just one.
```

Comment:
Just for type consistency

```suggestion
    Defaults to `1`.
```

Reply:
I'm removing the suggestion of the defaults from the vmware-iso docs as one of @SwampDragons suggestions for this pr (commit 61ee3a44f5365032549e684f23cc10ab7daff1e3) was to not explicitly set the cpus and memory size whilst instead relying on the user's default settings.

The other builders (both virtualbox and parallels) included a default of 1 cpu and 512 memory defined elsewhere (which was where I got those numbers from) and so those particualr docs specify the defaults. Just fyi.

Reply:
Gotcha ! 👍